addons/l10n_uy/data/template/account.account-uy.csv

```diff
@@ -71,8 +72,9 @@
 "uy_code_21320","Other debts","21320","liability_current","False","Otras deudas"
 "uy_code_21401","Minimum Sales Tax","21401","liability_current","False","Iva Ventas Mínima"
 "uy_code_21402","Basic Sales Tax","21402","liability_current","False","Iva Ventas Básica"
-"uy_code_21403","Tax to be Paid","21403","liability_payable","True","Iva a Pagar"
-"uy_code_21404","Withholding Tax","21404","liability_current","False","Iva Retenido"
+"uy_code_21403","Exempt Sales Tax","21403","liability_current","False","Iva Ventas Exento"
+"uy_code_21404","Tax to be Paid","21404","liability_payable","True","Iva a Pagar"
+"uy_code_21405","Withholding Tax","21405","liability_current","False","Iva Retenido"
```

Comment:
```suggestion
"uy_code_21405","VAT Withholding","21405","liability_current","False","Iva Retenido"
```
At least, this would make the a distinction with the Withholding Tax below